src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar calendar__month-length--28 calendar__day-start--tue page__calendar">
```

Comment:
Look at item 2 in checklist. Use correct naming for block modifier.

Reply:
Fixed